collector/src/bin/rustc-perf-collector/execute.rs

```diff
@@ -471,6 +474,13 @@ impl<'a> Processor for ProfileProcessor<'a> {
             path
         };
 
+        // Write text output to a file.
+        let write_text_to_file = |file, text| {
```

Comment:
Can probably just use https://doc.rust-lang.org/std/fs/fn.write.html.

Reply:
Hmm, I recall the `flush` being necessary... but local testing suggests that it isn't. Ok, I'll switch to `fs::write`.